tests/integration/changed-yum-conf/ansible/change_yum_conf.yml

```diff
@@ -0,0 +1,5 @@
+- hosts: all
+  tasks:
+
+    - name: Edit /etc/yum.conf
+      shell: echo '#random text' >> /etc/yum.conf
```

Comment:
Nit: I think it will be more clear, and simple if you do this inside the pytest with help of shell fixture, like for example here:
https://github.com/oamg/convert2rhel/blob/main/tests/integration/remove-excluded-pkgs/test_remove_excluded_pkgs.py#L15-L20.